go/client/cmd_chat_api.go

```diff
@@ -150,7 +169,7 @@ func (c *CmdChatAPI) ListV1(ctx context.Context) Reply {
 			},
 		}
 	}
-
+	cl.RateLimits.RateLimits = c.aggRateLimits(rlimits)
```

Comment:
I thought writing the whole thing out might make it more obvious if someone wanted to add an additional service RPC in here (that they would need to modify `rlimits`). But yeah, it could be written like you said, since there is just one call.